data_adapter/preprocessing.py

```diff
@@ -287,6 +287,7 @@ def __unpack_bandwidths(self, df: pd.DataFrame) -> pd.DataFrame:
     @staticmethod
     def __filter_subprocess(df: pd.DataFrame, subprocess: str) -> pd.DataFrame:
         df = df[df["type"] == subprocess]
+        df = df.dropna(axis=1)
```

Comment:
Two things:
1. `__filter_subprocess` is only called in case of multiple types per table - i think you want to have it in any case
2. As you did not define `how` parameter, column is dropped in case any value in column is NAN - but this would drop valid values too. You should change this to `how="all"` IMO

Reply:
I wanted to only filter the columns for multi type tables as they are the ones that could hold to many columns. If other tables provided containing only `None` columns I think the table does not follow the datamodel as supposed?

Thank you for noticing! Will change! :)